### ISSUE TITLE
Add markdown rendering in Instructor's message box

### ISSUE THREAD

**Issue by bkayes** [mterwill/office-hours-help-queue#179]
I'd like to have markdown rendering in the instructors message box so that we can embed links and bold certain important items.

**Comment by mterwill** [comment]
Makes sense. Maybe something like https://github.com/markedjs/marked?

**Comment by pl98** [comment]
Hi @mterwill! @annieli555 and I are interested in working on this issue, is it still available?

**Comment by mterwill** [comment]
Yes!
